src/base/bittorrent/session.cpp

```diff
@@ -1626,20 +1642,20 @@ bool Session::deleteTorrent(const QString &hash, const bool deleteLocalFiles)
     emit torrentAboutToBeRemoved(torrent);
 
     // Remove it from session
-    if (deleteLocalFiles) {
+    if (deleteOption != Torrent) {
```

Comment:
I use `if (deleteOption == Torrent)` here and switch the conditional bodies.

Reply:
Ok swapped.